src/Etg.Yams.Client/Etg.Yams.nuspec

```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
What happened to this file?

Reply:
I removed it as the batch file uses `Etg.Yams.Client.nuspec`. There were 2 files in the folder.

Reply:
Makes sense!